tapis-project/tapis-ui
# task/TUI-126-- react query mutations

## Overview: ##

useLogin mutation accepts a single parameter, login helper function accepts parameter list

## Related Github Issues: ##

* [TUI-126](https://github.com/tapis-project/tapis-ui/issues/126)

## Summary of Changes: ##

- login helper accepts username, password, basepath
- login hook from useLogin accepts username, password (does not accept onSuccess and onError callbacks anymore)
- Login component no longer accepts callbacks
- Apps/Login checks useTapisConfig login state for login status, rather than a callback which renders redirect

## Testing Steps: ##
1. npm start
2. make sure you are logged out
3. log in, make sure you are redirected to dashboard
4. log out

## UI Photos:

## Notes: ##
